From 1fd7a892170b3910bb1022b48b5c6e4d438e1a6d Mon Sep 17 00:00:00 2001
From: Maksim Rakitin <mrakitin@bnl.gov>
Date: Sat, 25 Sep 2021 09:45:19 -0400
Subject: [PATCH] fix-cl-platform-check-on-win

---
 xrt/backends/raycing/myopencl.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xrt/backends/raycing/myopencl.py b/xrt/backends/raycing/myopencl.py
index 4795d7e..74c0ca5 100644
--- a/xrt/backends/raycing/myopencl.py
+++ b/xrt/backends/raycing/myopencl.py
@@ -16,7 +16,7 @@ try:
     cl.get_platforms()
     os.environ['PYOPENCL_COMPILER_OUTPUT'] = '1'
     isOpenCL = True
-except ImportError:
+except Exception:
     isOpenCL = False
 
 try:
-- 
2.16.1

